### PR TITLE
Include UI as static files during build package build.

### PR DESCRIPTION
**Pull request by newswangerd**
- [x] Add the compiled UI to the build process
- [x] Automatic releases to PyPI

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,6 +23,28 @@
 ]
 
 
+class BuildPyCommand(setuptools.command.build_py.build_py):
+    """Custom build command."""
+
+    def run(self):
+        print('Downloading UI static files')
+
+        download_url = 'https://github.com/ansible/ansible-hub-ui/' + \
```
Comment (cutwater) [inline setup.py:32]: Please move `download_url` outside the function (class level \ module level) and make it a constant.

**Comment by newswangerd** [comment]
Temporarily closing to prevent travis from running while I test travis releases on my personal fork.

File: setup.py [hunk 2]
```diff
@@ -20,9 +24,32 @@
...
+            path='galaxy_ng/app/static/galaxy_ng'
+        )
+
+        setuptools.command.build_py.build_py.run(self)
+
+
 setup(
     name="galaxy-ng",
-    version="0.1.0a1",
+    version=version.get_git_version(),
```
Comment (cutwater) [inline setup.py:52]: I'm not sure that using git for versioning is a good idea, though it's acceptable for now.

File: setup.py [hunk 3]
```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env python3
 
 from setuptools import find_packages, setup
+import setuptools.command.build_py
+import tarfile
+import urllib.request
```
Comment (cutwater) [inline setup.py:6]: Minor: Please organize imports according to PEP8.

File: Dockerfile [hunk 4]
```diff
@@ -27,6 +27,7 @@ RUN set -ex; \
         python3-devel \
         libpq \
         libpq-devel \
+        git \
```
Comment (cutwater) [inline Dockerfile:30]: According to our last discussion `git` is not going to be used from within container. Is this dependency still required?

Comment (newswangerd) [reply Dockerfile:30]: Yes. The script throws an error if git isn't available that's different from the one if git is run outside of a repo, and I don't think it's a good idea to ignore that error.

**Comment by newswangerd** [comment]
The travis build for this pr doesn't seem to be getting linked here correctly, but it is passing

https://travis-ci.com/github/ansible/galaxy_ng/builds/159633401

**Pull request merged**